sdk/monitor/Azure.Monitor.OpenTelemetry.AspNetCore/src/OpenTelemetryBuilderExtensions.cs

```diff
@@ -211,6 +213,10 @@ public static OpenTelemetryBuilder UseAzureMonitor(this OpenTelemetryBuilder bui
                     }
                 });
 
+            // Enable custom events and set filter to enable collection.
+            builder.Services.AddSingleton<IApplicationInsightsEventLogger, ApplicationInsightsEventLogger>();
+            builder.Services.AddLogging(o => o.AddFilter(EventLoggerName, LogLevel.Information));
```

Comment:
Could you validate with unittest that user can override this filter, to achieve the effect of disabling EventTelemetry completely, if they chose to?